build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.4.1'
```

Comment:
Updating gradle is good, but I am doing this in the dev branch, not here. I don't want to break the build.